test/manifests/fakerp/admin-update.yaml

```diff
@@ -0,0 +1,3 @@
+# ADMIN_MANIFEST=test/manifests/fakerp/admin-update.yaml hack/upgrade.sh
```

Comment:
Separate test in the release or where do you see those?

Reply:
we need to discuss the exact plan - lots is needed here - this is just a dumb thing to help command line testing